### PR TITLE
Filter undefined copyright from Copyrights component.

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/Copyright/Copyright.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ function Copyright({ layerService, format, ...other }) {
         layerService
           .getLayersAsFlatArray()
           .filter(l => l.getVisible())
-          .map(l => l.getCopyright()),
+          .map(l => l.getCopyright())
+          .filter(cr => cr !== undefined),
```
Comment (gevos) [inline src/components/Copyright/Copyright.js]:
```suggestion
          .filter(cr => cr),
```
The copyright can be `"",` e.g. when it comes from Mapbox source and `sourceCache.used` is `false` see https://github.com/geops/react-spatial/pull/418

**Comment by gevos** [comment]
@edellucien Please, also have a look at https://github.com/geops/trafimage-maps/pull/408

File: src/components/Copyright/Copyright.js (hunk 1 above)
Comment (edellucien) [reply src/components/Copyright/Copyright.js]: So you want to filer on '' and false as well?

Comment (gevos) [reply src/components/Copyright/Copyright.js]: @edellucien Yes, I want to filter out `""`. And in which case is it `false`?

**Pull request merged**